### PR TITLE
update to Summer16 eID

### PR DESCRIPTION
**Pull request by OlivierBondu**
as per
https://twiki.cern.ch/twiki/bin/view/CMS/CutBasedElectronIdentificationRun2

I'm guessing I may have to update reference trees ? let's try and see...

for updating https://github.com/cp3-llbb/Framework/issues/183

### CODE REVIEW

File: python/ElectronsProducer.py [hunk 1]
```diff
@@ -9,10 +9,10 @@
             ea_R03 = cms.untracked.FileInPath('RecoEgamma/ElectronIdentification/data/PHYS14/effAreaElectrons_cone03_pfNeuHadronsAndPhotons.txt'),
```
Comment (blinkseb) [inline python/ElectronsProducer.py]: Can you also update this file to `RecoEgamma/ElectronIdentification/data/Summer16/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_80X.txt`?

**Review by blinkseb — CHANGES_REQUESTED** [review]
Thanks a lot for this! Jenkins test is actually ok, I'm not sure why the status was not updated here...

**Comment by OlivierBondu** [comment]
updated !

**Comment by OlivierBondu** [comment]
please test

**Comment by blinkseb** [comment]
Hm, something is wrong, tests should have failed, unless you updated the ref trees after your last commit? (history suggest it's not the case). One possibility is that the EA factors are actually the same between 7.6 and 8.0. If not, then there's something weird going on. Can you try locally and see if the computed isolation and EA factors for the electrons change from 7.6 to 8.0?

**Comment by OlivierBondu** [comment]
yes sorry, the commit message is not clear, but I did update the ref trees in the last commit, see https://github.com/cp3-llbb/Framework/pull/193/commits/d955db00d089b07cb62666daed8d9c7dce1f8fda

**Comment by blinkseb** [comment]
Ah cool! Thanks :)

**Review by blinkseb — APPROVED** [review]

**Pull request merged**